os/root-on-raid.md

```diff
@@ -0,0 +1,58 @@
+# Setting up Root on Raid
+
+Partitions used in RAID arrays need to have the gpt partition type uuid `be9067b9-ea49-4f15-b4f6-f36f8c9e1818` in order to boot. Because Ignition does not current support modifying existing partitions
+without wiping the partition table, the default ROOT partition cannot be used since it has different partition type uuid. In order to have the root partition in a raid array:
```

Comment:
[Paragraphs should not be linewrapped.](https://github.com/coreos/docs/blob/master/STYLE.md#line-wrapping-considered-harmful)